chain/chain/src/store/mod.rs

```diff
@@ -234,23 +235,23 @@ pub trait ChainStoreAccess {
             }
 
             let prev_hash = header.prev_hash();
-            let shard_layout = epoch_manager.get_shard_layout_from_prev_block(&block_hash)?;
-            let prev_shard_layout = epoch_manager.get_shard_layout_from_prev_block(prev_hash)?;
+            let shard_layout = epoch_manager.get_shard_layout_from_prev_block(prev_hash)?;
```

Comment:
nit: I'd try to keep naming consistent if possible

for instance `prev_shard_layout` and `current_shard_layout` for when there are 2 layouts in the same method
`shard_layout` when it's generic

Reply:
Right.